src/styles/page.scss

```diff
@@ -135,3 +135,7 @@
     color: inherit;
     opacity: 0.8;
 }
+
+.italic {
```

Comment:
We already have `fst-italic` (**f**ont-**st**yle: **italic**) from Bootstrap :)

Reply:
Nice, thanks!